starter_code/views/infos.hbs

```diff
@@ -0,0 +1,28 @@
+<div id="infos_container">
+    <div class="img_infos_container">
+
+        <img src="{{movie.image}}" alt="image">
+
+    </div>
+    <div class="info_movie">
+        <a href="/movies">Go back</a>
+
+        <h1 class="movie_titles">{{movie.title}}</h1>
+
+        <h2> Director: {{movie.director}} </h2>
+        <h3>Stars</h3>
+
+
+
+        <span>{{movie.stars}}</span>
+
+
+
+        <p> {{movie.description}} </p>
+
+        <span>{{movie.showtimes}}|</span>
```

Comment:
movie.showtime is an array so normally you should parse it with a forEach. Thus you could have this design : 10.00 | 12.00|